src/test/unit/keysim-globalifier.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+if (
+    jQuery.browser.name === "msie" &&
+    jQuery.browser.versionNumber <= 8
+   ) {
+    // This browser does not properly implement
+    // `Object.prototype.defineProperty`.
+    // This is required by the external keyboard event simulation module,
+    // `keysim`.
+    window.skipKeyboardShortcutTests = true;
+} else {
+    window.Keysim = require("keysim");
+    window.skipKeyboardShortcutTests = false;
+}
```

Comment:
Some [progress was made](https://github.com/eventualbuddha/keysim.js/issues/7) in making Keysim work in IE7&8. IE8 compatibility seems close. Should we consider it blocking?